README.md

```diff
@@ -1,43 +1,29 @@
-[![Build Status](https://travis-ci.org/maju6406/snow_record.svg?branch=master)](https://travis-ci.org/maju6406/snow_record)
-[![Puppet Forge](https://img.shields.io/puppetforge/v/beersy/snow_record.svg)](https://forge.puppetlabs.com/beersy/snow_record)
+# servicenow_tasks
 
-# snow_record
-
-This task manage records in ServiceNow.
+Tasks for manipulating ServiceNow records
 
 #### Table of Contents
 
 1. [Description](#description)
-2. [Setup - The basics of getting started with snow_record](#setup)
+2. [Setup - The basics of getting started with servicenow_tasks](#setup)
 3. [Usage - Configuration options and additional functionality](#usage)
-4. [Examples](#examples)
-5. [Limitations - OS compatibility, etc.](#limitations)
-6. [Development - Guide for contributing to the module](#development)
 
 ## Description
 
-snow_record provides a series of tasks to interact with a ServiceNow Instance. The main 3 tasks included with this module:
-
-* create - Create a ServiceNow object
-* read - Get a ServiceNow object
-* update - Update a ServiceNow object
-
-There are also 2 additional tasks for interacting with Incidents.
+servicenow_tasks provides a series of tasks to interact with a ServiceNow Instance. The 4 main tasks included with this module:
 
-* create_incident - Create an Incident
-* resolve_incident - Resolve an Incident
+* create_record - Create a ServiceNow record
```

Comment:
Is listing these necessary? Couldn't they see them via `bolt task show`? Thinking in the future if we add new tasks, it might be annoying to have to remember to update a README.

Reply:
It's only four things and it could be nice to see whats available from the README without installing the module. Just my 2 cents.

Reply:
Yeah, I don't have strong feelings. I just personally hate updating stuff unless I really have to.